src/client/Microsoft.Identity.Client/PoP/HttpPopClaimTypes.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Microsoft.Identity.Client.PoP
+{
+    internal static class HttpPopClaimTypes
+    {
+        #region JSON keys for Http request
+
+        /// <summary>
+        /// Access token with response cnf
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string At = "at";
+
+        /// <summary>
+        /// Http method (GET or POST)
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string M = "m";
```

Comment:
suggestion:  since these are constants in our code, should we spell them out a bit more for readability?  For example, call this "Method" instead of "M"?